website-custom-views/src/content/development/data-fetching.mdx

```diff
@@ -391,8 +391,6 @@ const Channels = () => {
 
 # Advanced usage
 
-<!-- ## Using fetch policy -->
-<!-- ## Customizing Apollo Cache -->
```

Comment:
I've removed the comments as they mess up with the VSCode config. We can always add back later.